### PR TITLE
local setup guide -> installation guide

### PR DESCRIPTION
**Pull request by pb82**
@philbrookes local setup guide should become the installation guide. Removed the extra chapter and did some minor refactoring to better adhere to the docs conventions.

### CODE REVIEW

**Comment by aliok** [comment]
:eye:

**Comment by aliok** [comment]
@pb82 sent a PR against this PR branch for some minor improvements: https://github.com/pb82/mobile-docs/pull/2

I will add some comments for Paul once you merge my PR.

**Comment by pb82** [comment]
@aliok thanks, your changes are included now

File: modules/ROOT/pages/installing-mobile-services.adoc [hunk 1]
```diff
@@ -10,16 +10,139 @@ NOTE: {org-name} recommends running OpenShift using the method described here. H
...
-== Next Steps
+****
+You can download Docker CE for MacOS version 17.09.1 link:https://download.docker.com/mac/stable/19543/Docker.dmg[here].
+****
+
+[role="secondary"]
+.Linux
+
+****
+You can download Docker CE for Linux version 17.09.1 link:https://download.docker.com/linux/centos/7/x86_64/stable/Packages/docker-ce-17.09.1.ce-1.el7.centos.x86_64.rpm[here].
```
Comment (aliok) [inline modules/ROOT/pages/installing-mobile-services.adoc]: This link is only for CentOS and perhaps Fedora and RHEL.
I feel like we should note it down and say for other distros, go to Docker installation guide: https://docs.docker.com/install/

File: modules/ROOT/pages/installing-mobile-services.adoc [hunk 2]
```diff
@@ -10,16 +10,139 @@ NOTE: {org-name} recommends running OpenShift using the method described here. H
...
+are installed and are the right version.
++
+Once verified it prompts you for your Docker Hub login credentials, which
+must be valid for the installation to continue.
++
+The next set of values are optional and can be left with the default values:
++
+[source,bash]
+----
+DockerHub Tag (Defaults to {release-number}):
```
Comment (aliok) [inline modules/ROOT/pages/installing-mobile-services.adoc]: @finp `{release-number}` here is rendered as is in the code block. String interpolation doesn't work.

**Comment by pmacko1** [comment]
fyi https://docs.google.com/document/d/1B3vI-aoLkdhkaZKvZJvmlpTRA05BnHwmBUp1YpDQDqU/edit?ts=5b27b293

File: modules/ROOT/pages/installing-mobile-services.adoc [hunk 3]
```diff
@@ -10,16 +10,149 @@ NOTE: {org-name} recommends running OpenShift using the method described here. H
...
 == Next Steps
 
-* Provision a Mobile Service
+* xref:con_services.adoc[Provision] a Mobile Service
+
+* xref:apps.adoc[Register] a Mobile App
+
+== Troubleshooting
+
+=== Switching from a different Docker version on Fedora 28
```
Comment (pb82) [inline modules/ROOT/pages/installing-mobile-services.adoc]: @pmacko1 based on your document. @finp ok to have this section (Troubleshooting)?

File: modules/ROOT/pages/installing-mobile-services.adoc [hunk 4]
```diff
@@ -10,16 +10,149 @@ NOTE: {org-name} recommends running OpenShift using the method described here. H
...
+[role="secondary"]
+.Linux
+
+****
+You can download Docker CE version 17.09.1 for rpm based distros from link:https://download.docker.com/linux/centos/7/x86_64/stable/Packages/docker-ce-17.09.1.ce-1.el7.centos.x86_64.rpm[here]. Otherwise have a look link:https://docs.docker.com/install[here].
+****
+
+== Local development
+
+. *Mobile-core Installer*
```
Comment (finp) [inline modules/ROOT/pages/installing-mobile-services.adoc]: how is this a task?

File: modules/ROOT/pages/installing-mobile-services.adoc [hunk 5]
```diff
@@ -10,16 +10,149 @@ NOTE: {org-name} recommends running OpenShift using the method described here. H
...
++
+We configure the local development installation of {org-name} {product-name} using ansible scripts in our link:https://github.com/aerogear/mobile-core[mobile-core] repo. Clone this repo to your local machine and check out the {release-number} tag using:
```
Comment (finp) [inline modules/ROOT/pages/installing-mobile-services.adoc]: we dont' do anything

File: modules/ROOT/pages/installing-mobile-services.adoc [hunk 6]
```diff
@@ -10,16 +10,149 @@ NOTE: {org-name} recommends running OpenShift using the method described here. H
...
++
+[source,bash,subs="attributes"]
+----
+git clone git@github.com:aerogear/mobile-core.git
+git checkout {release-number}
+----
+
+. *Installation*
```
Comment (finp) [inline modules/ROOT/pages/installing-mobile-services.adoc]: another non-step

File: modules/ROOT/pages/installing-mobile-services.adoc [hunk 7]
```diff
@@ -10,16 +10,149 @@ NOTE: {org-name} recommends running OpenShift using the method described here. H
...
++
+The installation process is straightforward when using the `install.sh`
```
Comment (finp) [inline modules/ROOT/pages/installing-mobile-services.adoc]: we should not claim it's straightforward, or claim any judgement about our offering

File: modules/ROOT/pages/installing-mobile-services.adoc [hunk 8]
```diff
@@ -10,16 +10,149 @@ NOTE: {org-name} recommends running OpenShift using the method described here. H
...
+    ]
+}
+
+PLAY RECAP ***********************************************************************************************************************************************************************************
+localhost                  : ok=44   changed=17   unreachable=0    failed=0
+----
++
+Which signifies that your installation has completed successfully.
+
+. *Verification*
```
Comment (finp) [inline modules/ROOT/pages/installing-mobile-services.adoc]: tense, step, opinions, avoid all these

.Verify the installation by :
.. Browse the following URL:
https://192.168.37.1:8443/console/
.. Log in using the following credentials

File: modules/ROOT/pages/installing-mobile-services.adoc [hunk 9]
```diff
@@ -10,16 +10,149 @@ NOTE: {org-name} recommends running OpenShift using the method described here. H
...
+----
++
+The developer login credentials are:
++
+----
+username: developer
+password: password
+----
++
+The admin login credentials are:
```
Comment (finp) [inline modules/ROOT/pages/installing-mobile-services.adoc]: what use case are we documenting that requires admin?

File: modules/ROOT/pages/installing-mobile-services.adoc [hunk 10]
```diff
@@ -10,16 +10,149 @@ NOTE: {org-name} recommends running OpenShift using the method described here. H
...
++
+----
+username: admin
+password: admin
+----
++
+Once logged in look for the "mobile" category across the top of the catalog package
```
Comment (finp) [inline modules/ROOT/pages/installing-mobile-services.adoc]: this is continuation of the verifying sub list
.. Click Add to Project and choose Browse Catalog from the options.
.. Check that there is a Mobile tab displayed.
.. If you do not see a Mobile tab, see Troubleshooting

File: modules/ROOT/pages/installing-mobile-services.adoc (hunk 9 above)
Comment (pb82) [reply modules/ROOT/pages/installing-mobile-services.adoc]: Hmm none really. Will remove this.

**Comment by pb82** [comment]
@finp Thanks for the review. I've addressed most of your comments and answered the other ones. Could you have another look?

File: modules/ROOT/pages/installing-mobile-services.adoc (hunk 10 above)
Comment (finp) [reply modules/ROOT/pages/installing-mobile-services.adoc]: I'll review again, leave it for now

**Pull request merged**